opendebates/moderator_views.py

```diff
@@ -60,6 +60,13 @@ def merge(request):
         votes_to_merge = Vote.objects.filter(submission=to_remove).exclude(
             voter__in=votes_already_cast)
 
+        previous_debate_time = get_previous_debate_time()
+        if previous_debate_time:
+            current_votes_to_merge = votes_to_merge.filter(
+                created_at__gt=previous_debate_time).count()
+        else:
+            current_votes_to_merge = 0
```

Comment:
Updated to do the full count, since I wanted all `current_votes` to be ready and up to date if and when the feature is enabled.